src/Workspaces/Core/Desktop/Workspace/MSBuild/MSBuildProjectLoader.cs

```diff
@@ -30,7 +30,7 @@ public class MSBuildProjectLoader
         private readonly Dictionary<string, string> _extensionToLanguageMap = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
 
         /// <summary>
-        /// Create a new instance of an <see cref="MSBuildProjectLoader"/>.
+        /// Create a new instance of a <see cref="MSBuildProjectLoader"/>.
```

Comment:
Technically, 'an' is correct since MS is pronounced "em ess", and that leads with a vowel sound... :smile:

(English is a terrible language.)